snap/README.md

```diff
@@ -260,11 +248,33 @@ sudo snap set edgexfoundry env.security-proxy.user=user01,USER_ID,ES256
 sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
 ```
 
-The second step is then to generate a token using the user ID which you specified:
+Alternatively, you can create the user using the secrets-config command. You need to provide the following:
+
+- The username
+- The public key
+- The API Gateway Admin JWT token
+- (optionally) ID. This is a unique string identifying the credential. It will be required in the next step to
+create the JWT token. If you don't specify it,
+then an autogenerated one will be output by the secrets-config command
+```bash
+
+# get API Gateway/Kong token
+JWT_FILE=/var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt
+JWT=`sudo cat ${JWT_FILE}`
+
+# use secrets-config to add user
+$ edgexfoundry.secrets-config proxy adduser --token-type jwt --user user01 --algorithm ES256 --public_key public.pem --id USER_ID --jwt ${JWT}
```

Comment:
Remove the `$` from the beginning for consistency.

Reply:
modified

> > @MonicaisHer , @farshidtz thanks for the review. I've updated the PR, including your suggestions and clarifying the examples.
> 
> Thanks. I added some inline comments for minor modifications.
> 
> I feel there are additional changes compared to before, but there is no way to know due to the lost history. In future, please avoid amending the commit unless you made a mistake which you need to go back and fix. Having separate commits helps the reviewers to quickly see the differences and possible new additions. Moreover, it helps to detect changes that have been committed by mistake.
> 
> In the end, we can still squash and merge to have a single commit on the main branch.

Thanks, I've added a new commit with suggested changes.